tools/convert_datasets/pascal_context.py

```diff
@@ -0,0 +1,86 @@
+import argparse
```

Comment:
We may add doc [here](https://github.com/open-mmlab/mmsegmentation/blob/master/docs/getting_started.md) about how to prepare pascal context dataset.